### PR TITLE
silk

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
## About The Pull Request
Spider webs got a buff of sorts, you can now harvest silk into the new ~~wood based ecom but more balanced~~ cash trade item that can even be given/sold to the faith as it has when broken down, biomatter. - Irl silk is a massive proteen so this is infact based in logic
To get this you must have the proper tool - A silk harvester wand
A death wish
And a spider farm or nest
## Changelog
:cl:
/:cl:

### CODE REVIEW

**Comment by MSieghardt** [comment]
Balance issue:
Spider Cht'Mants can weave webs themselves, making them a farm of infinite good money. They are also friendly to spider, making it a no-hassle abuse-able money farm.

**Review by KathyRyals — CHANGES_REQUESTED** [review]

File: code/modules/materials/materials.dm [hunk 1]
```diff
@@ -674,6 +674,15 @@ var/list/name_to_material
 	melting_point = T0C+300
 	flags = MATERIAL_PADDING
 
+/material/silk //todo
+	name = MATERIAL_SILK
+	stack_origin_tech = list(TECH_MATERIAL = 2)
+	composite_material = list(MATERIAL_BIOMATTER = 1) //So we have a vaule to more then one faction
```
[the remark this answers is not in the document]
Comment (Trilbyspaceclone) [reply code/modules/materials/materials.dm:680]: to the church

**Comment by Trilbyspaceclone** [comment]
@MSieghardt 
Spider farms are intended

**Comment by KathyRyals** [comment]
Won't merge until you find a way to prevent PC farming silk by spamming their web perk.

**Comment by Kazkin** [comment]
Aye, my suggestion is make a subtype of web that has value 0, biomatter 0, and a different name/description that the player perk generates instead of the standard webs. That way we don't have system gaming.

**Comment by Trilbyspaceclone** [comment]
give me a day or two then.

**Comment by KathyRyals** [comment]
Looks good.

**Pull request merged**